pjmedia/src/pjmedia/rtp.c

```diff
@@ -202,12 +202,12 @@ PJ_DEF(pj_status_t) pjmedia_rtp_decode_rtp2(
     }
 
     /* Check that offset is not greater than packet size */
-    if (offset > pkt_len) {
+    if (offset >= pkt_len) {
```

Comment:
This keeps going back and forth between `>` and `>=`
Please check https://github.com/pjsip/pjproject/pull/3283
I think the current version is already correct